parachain/src/command.rs

```diff
@@ -286,26 +287,105 @@ pub fn run() -> Result<()> {
+                                new_partial::<interlay_runtime::RuntimeApi, InterlayRuntimeExecutor>(&config)?;
+                            cmd.run(partials.client)
+                        })
+                    } else if chain_spec.is_kintsugi() {
+                        runner.sync_run(|config| {
+                            let partials =
+                                new_partial::<kintsugi_runtime::RuntimeApi, KintsugiRuntimeExecutor>(&config)?;
+                            cmd.run(partials.client)
+                        })
+                    } else if chain_spec.is_testnet() {
+                        runner.sync_run(|config| {
+                            let partials = new_partial::<testnet_runtime::RuntimeApi, TestnetRuntimeExecutor>(&config)?;
+                            cmd.run(partials.client)
+                        })
+                    } else {
+                        Err("Chain doesn't support benchmarking".into())
+                    }
+                }
+                BenchmarkCmd::Storage(cmd) => {
+                    let chain_spec = &runner.config().chain_spec;
+                    if chain_spec.is_interlay() {
+                        runner.sync_run(|config| {
+                            let partials =
+                                new_partial::<interlay_runtime::RuntimeApi, InterlayRuntimeExecutor>(&config)?;
+                            let db = partials.backend.expose_db();
+                            let storage = partials.backend.expose_storage();
+                            cmd.run(config, partials.client.clone(), db, storage)
+                        })
+                    } else if chain_spec.is_kintsugi() {
+                        runner.sync_run(|config| {
+                            let partials =
+                                new_partial::<kintsugi_runtime::RuntimeApi, KintsugiRuntimeExecutor>(&config)?;
+                            let db = partials.backend.expose_db();
+                            let storage = partials.backend.expose_storage();
+                            cmd.run(config, partials.client.clone(), db, storage)
+                        })
+                    } else if chain_spec.is_testnet() {
+                        runner.sync_run(|config| {
+                            let partials = new_partial::<testnet_runtime::RuntimeApi, TestnetRuntimeExecutor>(&config)?;
+                            let db = partials.backend.expose_db();
+                            let storage = partials.backend.expose_storage();
+                            cmd.run(config, partials.client.clone(), db, storage)
+                        })
+                    } else {
+                        Err("Chain doesn't support benchmarking".into())
+                    }
+                }
+                BenchmarkCmd::Overhead(_) => Err("Unsupported benchmarking command".into()),
             }
         }
+        // Some(Subcommand::Benchmark(cmd)) => {
```

Comment:
There will be conflicts now due to the `instant-seal` changes, also did you mean to uncomment or delete these lines?